src/js/state/migrations/202005181133_addTimeFormatPref.test.ts

```diff
@@ -1,12 +0,0 @@
-import Prefs from "../Prefs"
```

Comment:
Sure we want to remove this? For better or worse, I thought these migrations were effectively immutable and need to stay in the app as is.

Reply:
The migrations themselves shouldn't be touched, but this is a migration test file. I figure if this old migration has been working until now, we can delete the test. The test incorrectly relied on code that might change like the selector functions in Prefs. So rather than rewrite it, I just thought we could remove it .